docs/Kube-API-examples.md

```diff
@@ -10,3 +10,33 @@ You will likely need to adapt those for your own needs.
 * [Hive PullSecret Secret](crds/pullsecret.yaml)
 * [Hive ClusterDeployment](crds/clusterDeployment.yaml)
 * [Hive ClusterDeployment-SNO](crds/clusterDeployment-SNO.yaml)
+
+
+
+###Creating InstallConfig overrides
+
+In order to alter the default install config yaml used when running `openshift-install create` commands.
+[REST-API reference](user/guide/install-customization.md)
+
+Add an annotation with the desired options, the clusterdeployment controller will update the install config yaml with the annotation value.
```

Comment:
Presumably this has to be done prior to some point in the installation process. It would be a good idea to specify what that requirement is.